play-services-api/src/main/java/com/google/android/gms/semanticlocation/SemanticLocationEventRequest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-FileCopyrightText: 2023 microG Project Team
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.google.android.gms.semanticlocation;
+
+
+import android.os.Parcel;
+
+import androidx.annotation.NonNull;
+
+import com.google.android.gms.common.internal.safeparcel.AbstractSafeParcelable;
+import com.google.android.gms.common.internal.safeparcel.SafeParcelable;
+import com.google.android.gms.common.internal.safeparcel.SafeParcelableCreatorAndWriter;
+
+@SafeParcelable.Class
+public class SemanticLocationEventRequest extends AbstractSafeParcelable {
+    @Field(1)
+    public float position;
+
+    public SemanticLocationEventRequest(float position) {
```

Comment:
Add `@Constructor` to the constructor and `@Param(1)` to the first parameter to indicate to safe-parcel that the first parameter is used to populate the field annotated with `@Field(1)`. This allows removing the additional no-argument constructor.